src/services/api.ts

```diff
@@ -626,7 +629,7 @@ export const getTransactionHistory = async (
 
   const _config = config.get()
 
-  const url = new URL(`${_config.apiUrl}/analytics/transfers`)
+  const url = new URL(`${config.getApiUrl('v2')}/analytics/transfers`)
```

Comment:
Is this the only change for v2 of the analytics? Is the response the same?

Reply:
Yes, you can test with these urls

https://li.quest/v1/analytics/transfers?wallet=0xBD55C2F306C97Fd1d3E7A023f7c4834a2F472834&integrator=li.fi-playground

https://li.quest/v2/analytics/transfers?wallet=0xBD55C2F306C97Fd1d3E7A023f7c4834a2F472834&integrator=li.fi-playground